types/csso/csso-tests.ts

```diff
@@ -0,0 +1,40 @@
+import csso = require('csso');
```

Comment:
Since this csso uses standard ES module syntax, it's best to import it using the standard syntax `import * as csso from 'csso';`

Reply:
Hi @sandersn,

that's not true. If you look at [the source code](https://github.com/css/csso/blob/master/lib/index.js#L130) of `csso` and the official README you can see that it is a typical Node.js / CommonJS library 😉 

![d174f2c3-c011-4fb1-8d7c-f969bb4371c6](https://user-images.githubusercontent.com/149248/41677054-8aa1bcc8-74c7-11e8-823b-87eba5ad538f.png)

Reply:
Ah I got it now. I wrote the definition wrong 🤦‍♂️ 
Now I fixed it 🤓